src/multivariate/mvlognormal.jl

```diff
@@ -27,14 +27,17 @@ abstract type AbstractMvLogNormal <: ContinuousMultivariateDistribution end
 
 function insupport(::Type{D},x::AbstractVector{T}) where {T<:Real,D<:AbstractMvLogNormal}
     for i=1:length(x)
-      @inbounds 0.0<x[i]<Inf ? continue : (return false)
+        @inbounds 0.0<x[i]<Inf ? continue : (return false)
     end
-    true
+    return true
 end
+
 insupport(l::AbstractMvLogNormal,x::AbstractVector{T}) where {T<:Real} = insupport(typeof(l),x)
 assertinsupport(::Type{D},m::AbstractVector) where {D<:AbstractMvLogNormal} = @assert insupport(D,m) "Mean of LogNormal distribution should be strictly positive"
 
-###Internal functions to calculate scale and location for a desired average and covariance
+"""
```

Comment:
This isn't appropriate as the comment described two functions, not one.

Reply:
reversed in last commit